### DISCUSSION TITLE
How do you install python packages?

### DISCUSSION THREAD

**Discussion by VasilisThePikachu** [coder/code-server#3249] [Q&A]
pip and pip3 are not a commands that work and i can't install them with apt cause i have no sudo access

if it helps im running it in docker

**Comment by nooobcoder** [comment]
To install a package from a repository (here apt) you do need sudo rights @VasilisThePikachu.
`sudo apt install python3-pip`
For more information refer here, [](https://askubuntu.com/questions/339/how-can-i-install-a-package-without-root-access)

**However, you can try installing by downloading the python script and installing pip through it,**
`wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py --user`

Please post back any queries, if you have.

Thank You.

**Reply by VasilisThePikachu** [reply]
both result in sudo and wget not found, im running them in the terminal in vscode

**Reply by nooobcoder** [reply]
@VasilisThePikachu seems like impossible to me now, because wget is not installed. To install wget you need to use apt again  with sudo rights.
Try by curl, good luck
I guess this is a very rare case of installation of a package, and is not related to code-server at all. code-server is just a docker container that is a debian based host that serves vscode service through a web browser.

Cannot help you though in this case, sorry!
